### PR TITLE
Updated descriptions to use better cli ref

### PR DESCRIPTION
**Pull request by FrenchBen**
### Proposed changes
Update CLI descriptions to use the proper CLI references

### Related issues (optional)
https://github.com/docker/docker/pull/30514

### CODE REVIEW

**Comment by mstanleyjones** [comment]
Building this locally to check it now. I think we can ignore the Netlify error as it doesn't actually run the Dockerfile.

File: _data/engine-cli/docker_stats.yaml [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
+  The following example uses a template without headers and outputs the
+  `Container` and `CPUPerc` entries separated by a colon for all images:
+
+  ```bash
+  $ docker stats --format "{{.Container}}: {{.CPUPerc}}"
+
+  09d3bb5b1604: 6.61%!
+(MISSING)  9db7aa4d986d: 9.19%!
+(MISSING)  3f214c61ad1d: 0.00%!
+(MISSING)  ```
```
Comment (mstanleyjones) [inline _data/engine-cli/docker_stats.yaml]: This line causes the following error when trying to build the Dockerfile:

```none
jekyll 3.3.1 | Error:  (/usr/src/app/allv/_data/engine-cli/docker_stats.yaml): could not find expected ':' while scanning a simple key at line 81 column 1
```

To test, build the Dockerfile in your branch with:

```bash
$ docker build -t test_the_thing .
```

Comment (FrenchBen) [inline _data/engine-cli/docker_stats.yaml]: yup - working on fixing this ugly output - Not sure why the `(MISSING)` bits are being added 😞

**Comment by FrenchBen** [comment]
@mstanleyjones @johndmulhausen I can haz LGTM and get this merged?

**Pull request merged**